src/com/homepractice/OverridingConcept.java

```diff
@@ -0,0 +1,36 @@
+package com.homepractice;
+//method name and signature must be same ...return type can be different from version 1.5 onwords
+//but it should be child type i.e Object (return type of parent method) then String (return type of 
+ // child method) is allowed
+// You can write private method with same name and arg in both child and parent class but it is not 
+// overriding
+// you can not override final method
+// can not reduce scope of access modifier in child class .. but can be increased 
```

Comment:
why?

Reply:
suppose you have created reference as:
Parent p = new Child();
This will call child method..If you are reducing the access then parent ref can not access it.